eng/scripts/Create-APIView.ps1

```diff
@@ -111,15 +117,17 @@ function Invoke-SwaggerAPIViewParser {
       LogGroupEnd
 
       $generatedAPIViewTokenFile = Get-ChildItem -File | Select-Object -First 1
-      $readMeTag = $generatedAPIViewTokenFile.BaseName
-
-      LogSuccess " Generated '$Type' APIView Token File using file, '$readMeFile' and tag '$readMeTag'"
 
-      $apiViewTokensFilePath = [System.IO.Path]::Combine($TokenDirectory, "$resourceProvider.$Type.json")
-      LogInfo " Moving generated APIView Token file to '$apiViewTokensFilePath'"
-      Move-Item -Path $generatedAPIViewTokenFile.FullName -Destination $apiViewTokensFilePath -Force > $null
+      if (Test-Path -Path $generatedAPIViewTokenFile) {
```

Comment:
If this file doesn't exist I would assume the parser failed. Wouldn't we have a failing exit code to look at in that case?

Reply:
No necessarily. In the case where the tag is not found in the readme.md it will not create any APIView token.